### DISCUSSION TITLE
Warn when rules.yml are not found

### DISCUSSION THREAD

**Discussion by Rodeoclash** [ory/oathkeeper#1109] [Ideas]
I have a reasonably complicated setup where my `rules.yml` file is templated in Terraform then downloaded to the container using a sidecar. Or, it would be had I remembered to configure the sidecar to actually download the file.

Perhaps we should log a warning if a `rules.yml` is specified but not found.

**Comment by vinckr** [comment]
Hey Samuel, 
this sounds like a reasonable addition to me.
Feel free to create a design document or pull request if you are willing to work on this.
